1-org/envs/shared/essential_contacts.tf

```diff
@@ -16,18 +16,17 @@
 
 locals {
   gcp_scc_admin         = var.gcp_groups.scc_admin == null ? local.group_org_admins : var.gcp_groups.scc_admin
-  gcp_platform_viewer   = var.gcp_groups.platform_viewer == null ? local.group_org_admins : var.gcp_groups.platform_viewer
   gcp_security_reviewer = var.gcp_groups.security_reviewer == null ? local.group_org_admins : var.gcp_groups.security_reviewer
   gcp_network_viewer    = var.gcp_groups.network_viewer == null ? local.group_org_admins : var.gcp_groups.network_viewer
 
   # Notification categories details: https://cloud.google.com/resource-manager/docs/managing-notification-contacts#notification-categories
   categories_map = {
     "BILLING"         = setunion([local.group_billing_admins, var.billing_data_users])
     "LEGAL"           = setunion([local.group_org_admins, var.audit_data_users])
-    "PRODUCT_UPDATES" = setunion([local.gcp_scc_admin, local.gcp_platform_viewer])
+    "PRODUCT_UPDATES" = [local.gcp_scc_admin]
```

Comment:
Can you clarify for me please where in the documentation did you use as reference to update this file? I didn't fully get it this part.

Reply:
I'm using [Foundations docs](https://cloud.google.com/architecture/security-foundations/authentication-authorization#groups_for_access_control) as reference. And in this doc there's no Platform Viewer group.
Also, on [Elliot's spreadsheet](https://docs.google.com/spreadsheets/d/1KztzTlFZnq4OLnAr4TTqdpFATWEUOXGfTmoYO55gfS0/edit?resourcekey=0-hJF7wFZkkZdrJLJDlAbrtQ#gid=2133213467&range=A8:H8) it says that this groups is an `anti-pattern`.